experiments/Makefile

```diff
@@ -1,12 +1,10 @@
+# This file contains the benchmarking script configurations for various near neighbor datasets
+# The main configurable parameter is the type of index: --index-type (hnsw, flatnav)
+# We use dataset-name for the name of the plots.
```

Comment:
We should also mention that one can (and should) tweak the number of build threads based on their runtime environment. It doesn't have to be 16. It can be less or more.